### PR TITLE
[flutter_tools] HACKTOBERFEST

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

Fix typos

### CODE REVIEW

File: packages/flutter_tools/lib/src/web/chrome.dart [hunk 1]
```diff
@@ -162,13 +162,13 @@ class ChromiumLauncher {
 
   /// Launch a Chromium browser to a particular `host` page.
   ///
-  /// `headless` defaults to false, and controls whether we open a headless or
-  /// a `headfull` browser.
+  /// [headless] defaults to false, and controls whether we open a headless or
+  /// a "headfull" browser.
```
Comment (shihaohong) [inline packages/flutter_tools/lib/src/web/chrome.dart:166]: [Headful](https://www.merriam-webster.com/dictionary/headful#:~:text=%3A%20a%20quantity%20(as%20of%20information)%20that%20fills%20the%20head)? I have no idea :)

**Review by shihaohong — APPROVED** [review]
LGTM

File: packages/flutter_tools/lib/src/web/chrome.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/web/chrome.dart:166]: not headless .. heady? 🤷‍♂️ I did write this after all

**Comment by jonahwilliams** [comment]
Google testing issue appears to be a rebase problem? this PR did not rename that class

**Pull request merged**